<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectomeInspectionDialog</class>
 <widget class="QDialog" name="ConnectomeInspectionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connectome Analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTreeWidget" name="SubjectsTree">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Quality control preview</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Diffusion weighted b0</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="b0CoronalLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="b0SagittalLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="b0AxialLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Labels map</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="LabelmapCoronalLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LabelmapSagittalLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LabelmapAxialLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Skull stripped T1</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="T1CoronalLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="T1SagittalLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="T1AxialLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>125</horstretch>
                <verstretch>125</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder>label_10</zorder>
         <zorder></zorder>
         <zorder>label_11</zorder>
         <zorder>label_12</zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Orientation correction</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <widget class="QRadioButton" name="NoChangeRadio">
            <property name="text">
             <string>No change/reset</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="minimumSize">
               <size>
                <width>314</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="margin">
                <number>5</number>
               </property>
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="InvertXRadio">
                   <property name="text">
                    <string>Invert X</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QRadioButton" name="InvertYRadio">
                   <property name="text">
                    <string>Invert Y</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QRadioButton" name="InvertZRadio">
                   <property name="text">
                    <string>Invert Z</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="SwapXYRadio">
                   <property name="text">
                    <string>Swap XY</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QRadioButton" name="SwapXZRadio">
                   <property name="text">
                    <string>Swap XZ</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QRadioButton" name="SwapYZRadio">
                   <property name="text">
                    <string>Swap YZ</string>
                   </property>
                   <property name="autoExclusive">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectomeInspectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectomeInspectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
